deploy-board/deploy_board/templates/configs/config_map.tmpl

```diff
@@ -42,6 +42,30 @@
     </div>
 </div>
 
+<div class="modal fade" id="saveEntryModalId" tabindex="-1" role="dialog"
+     aria-labelledby="saveEntryModalLabel" aria-hidden="true">
+    <div class="modal-dialog modal-lg">
+        <div class="modal-content">
+            <form id="saveEntryFormId" class="form-horizontal" method="post" role="form">
+                <div class="modal-header">
+                    <button type="button" class="close" data-dismiss="modal"><span
+                            aria-hidden="true">&times;</span><span class="sr-only">Close</span>
+                    </button>
+                    <h4 class="modal-title" id="saveEnvModalLabel">Save New {{ panelTitle }}</h4>
+                </div>
+                <div class="modal-body">
+                    Are you sure you want to <strong>save new config</strong> for this cluster?<br>
+                    Please double check the new added config with red color before saving. Thanks.
```

Comment:
red color means new entries or error entries? If it's new probably we should have them in different color. Red often means error I guess.

Reply:
new entries. what's the best color for new entries on your mind please?

Reply:
not an UX expert but probably blue green or yellow.

Reply:
ok, used yellow ^^